lib/linguist.rb

```diff
@@ -53,7 +54,7 @@ def detect(blob, allow_empty: false)
   #
   #   blob - An object that quacks like a blob.
   #   languages - An Array of candidate Language objects that were returned by the
-  #               previous strategy.
+  #               previous strategy.
```

Comment:
Uh? What happened here?

Reply:
Looks like a pair of non-breaking spaces (U+00A0) were inserted at some point (probably be a macOS user who accidentally held the <kbd>Option</kbd> key while hitting the spacebar. Doing that inserts `\xA0` instead of `\x20`).

My editors are setup to convert those nasty little bastards to ordinary spaces when saving source code. So I must've unknowingly fixed a minor formatting error. 😜